src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,53 @@
+.calendar {
+  display: flex;
+  justify-content: center;
+
+  &__month {
+    display: flex;
+    gap: $gap-between-days;
+    flex-wrap: wrap;
+    max-width: calendarWidth(
+      $day-size,
+      $gap-between-days,
+      $calendar-side-paddings
+    );
+    padding: 20px $calendar-side-paddings;
```

Comment:
Don't be greedy)). 20px is normal, but can be more. Also, you can use only padding-top or margin-top. I would prefer to use a margin-top.

Reply:
generally agree! but here I decided to add padding 20px on the top and bottom and 10px - left and right; in case if I will use it again it will contain necessary paddings, but I'm not sure... maybe I'm wrong